### PR TITLE
Enh hints

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

**Comment by danielballan** [comment]
Glorious!

File: ophyd/quadem.py [hunk 1]
```diff
@@ -92,6 +92,36 @@ def __init__(self, *args, **kwargs):
...
+            return {'fields': [getattr(self, c).mean_value.name
+                               for c in ['current1', 'current2',
+                                         'current3', 'current4']]}
+        return self._hints
+
+    @hints.setter
+    def hints(self, val):
+        self._hints = val if val is None else dict(val)
+
+    def wtf(self):
```
Comment (klauer) [inline ophyd/quadem.py]: 👍

Comment (tacaswell) [reply ophyd/quadem.py]: That should not have made it into a commit....

Comment (danielballan) [reply ophyd/quadem.py]: The PR reviewer/merger sniggers, regretfully.

**Pull request merged**